fe/fe-core/src/main/java/org/apache/doris/catalog/Column.java

```diff
@@ -846,6 +862,10 @@ public void checkSchemaChangeAllowed(Column other) throws DdlException {
             return;
         }
         // TODO check cluster key
+
+        if (generatedColumnInfo != null || other.getGeneratedColumnInfo() != null) {
+            throw new DdlException("Temporarily not supporting alter table modify generated columns.");
```

Comment:
not use Temporarily in error message
```suggestion
            throw new DdlException("not supporting alter table modify generated columns.");
```